vernac/declare.ml

```diff
@@ -2086,9 +2101,9 @@ let check_single_entry { entries; uctx } label =
   | _ ->
     CErrors.anomaly ~label Pp.(str "close_proof returned more than one proof term")
 
-let finalize_proof ~pm proof_obj proof_info =
+let finalize_proof ~pm proof_obj ending proof_info =
```

Comment:
This new parameter seems strange, proof_info already contains an ending, why the need to pass another one?